hugr/Cargo.toml

```diff
@@ -52,6 +54,9 @@ delegate = "0.12.0"
 paste = "1.0"
 strum = "0.26.1"
 strum_macros = "0.26.1"
+proptest = { version =  "1.4.0", optional = true }
```

Comment:
Presumably these are used only if testing, so can they not go in `dev-dependencies`?

Reply:
Ah got it. So these are the `"dep: xxx"` on the RHS of `proptest` in `[features]`